contracts/Commitment/WhitelistedCommitment.sol

```diff
@@ -1,8 +1,10 @@
 pragma solidity 0.4.15;
 
 import './PublicCommitment.sol';
+import '../Reclaimable.sol';
 
-contract WhitelistedCommitment is AccessControlled, AccessRoles, CommitmentBase {
+
+contract WhitelistedCommitment is AccessControlled, AccessRoles, CommitmentBase, Reclaimable {
```

Comment:
It should not allow to reclaim `Neumark` token before WhitelistedCommitment is finalized,  Neumarks are already handled differently
* `abortCommitment()` will burn all neumarks in possession of the contract if called before `startDate`
* no one should be able to reclaim or burn Neumarks when commitment is in progress

use `isFinalized()` function to get finalization status
All neumarks in contract possession after finalization can be reclaimed